concordium-sdk/src/main/java/com/concordium/sdk/transactions/TransferScheduleWithMemo.java

```diff
@@ -0,0 +1,82 @@
+package com.concordium.sdk.transactions;
+
+import com.concordium.sdk.types.UInt64;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import lombok.val;
+
+import java.nio.ByteBuffer;
+
+/**
+ * Transfer an amount with schedule and memo.
+ */
+@ToString
+@Getter
+@EqualsAndHashCode(callSuper = true)
+public final class TransferScheduleWithMemo extends Payload {
+    /**
+     * The account address of the recepient.
+     */
+    private final AccountAddress to;
+    /**
+     * The release schedule. This can be at most 255 elements.
+     */
+    private final Schedule[] amount;
+    private final Memo memo;
+
+    private final UInt64 maxEnergyCost;
```

Comment:
Same comment as above. 
The price should be calculated before. It is known before the execution.

Reply:
changed